<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-N60QB27" failures="1" tests="3" name="TestCase" time="142.63" errors="0" timestamp="2022-12-12T15:19:15 IST">
  <testcase classname="tests.TestCase" name="TestCase3" time="0.055"/>
  <testcase classname="tests.TestCase" name="TestCase2" time="0.005"/>
  <testcase classname="tests.TestCase" name="TestCase1" time="27.505">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at tests.TestCase.TestCase1(TestCase.java:17)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- TestCase1 -->
</testsuite> <!-- TestCase -->
